blockchain/vm/jump_table.go

```diff
@@ -401,11 +402,11 @@ func newFrontierInstructionSet() JumpTable {
 		},
 		BALANCE: {
 			execute:         opBalance,
-			constantGas:     params.BalanceGas,
+			constantGas:     params.BalanceGasEIP150,
```

Comment:
@yoomee1313 
Why do you change newFrontierInstructionSet for istanbul hardfork?